test/pgident.conf

```
@@ -0,0 +1,6 @@
+# PostgreSQL User Name Maps
+# =========================
+
+# MAPNAME       SYSTEM-USERNAME         PG-USERNAME
+test		pgbouncer.acme.org	someuser
```

I think it would be good to test a case where there are multiple mappings for the same name, then the cert should be able to authenticate as both:
```suggestion
test		pgbouncer.acme.org	someuser
test		pgbouncer.acme.org	someuser2
```

---

The current design does not support this.  The first encountered map is effective. But I see that postgres supports multiple mappings with the same name:
https://www.postgresql.org/docs/current/auth-username-maps.html#:~:text=%23%20MAPNAME%20%20%20%20%20%20%20SYSTEM%2DUSERNAME,omicron%20%20%20%20%20%20%20%20%20bryanh%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20guest1

Need to make data structure changes to support this.